steps/fenceAPI.js

```diff
@@ -2,12 +2,11 @@
 
 let username = process.env.INDEX_USERNAME;
 let password = process.env.INDEX_PASSWORD;
-let accessTokenHeaders = {
-  'Accept': 'application/json',
-  'Authorization': `bearer ${process.env.ACCESS_TOKEN}`
-};
 let indexAuth = Buffer.from(`${username}:${password}`).toString('base64');
```

Comment:
check how did username/password are used here. Remove repeated code, update the util function